charts/postgres/templates/initjob.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: postgres-init
+  namespace: telemetry
+spec:
+  template:
+    spec:
+      containers:
+        - name: init-db
+          image: postgres:16
+          command:
+            - sh
+            - -c
+            - |
+              until pg_isready -h postgres -U postgres; do sleep 1; done
+              psql -h postgres -U postgres -f /scripts/init.sql
```

Comment:
Potentially we could use something like https://stackoverflow.com/a/17757560/8629780 to check for the existence of the DBs, skipping the init.sql if they already exist?

Or perhaps better we could follow the "Workaround from with psql" (first approach) in the accepted answer in https://stackoverflow.com/questions/18389124/simulate-create-database-if-not-exists-for-postgresql.

That would make the init.sql idempotent ;-)